ocaml/xapi/cli_operations.ml

```diff
@@ -4798,15 +4798,20 @@ let update_introduce printer rpc session_id params =
   let uuid = Client.Pool_update.get_uuid ~rpc ~session_id ~self:update in
   printer (Cli_printer.PList [uuid])
 
+let livepatch_status_to_string state =
+  match state with
+  | `ok -> "ok"
+  | `ok_livepatch_complete -> "ok_livepatch_complete"
+  | `ok_livepatch_incomplete -> "ok_livepatch_incomplete"
```

Comment:
I think we ought to explain these rather than just print out the text of the enum. Perhaps:

"Ok."
"Ok: Patch can be applied without reboot."
"Ok: Patch can be applied, but a reboot will be required."